src/ArgParse.jl

```diff
@@ -1,10 +1,9 @@
-isdefined(:OptionsMod) || require("Options")
+isdefined(:OptionsMod) || @eval import Options
```

Comment:
Thanks for the links to those conversations. I put in a cleaner pull
request with just the @eval change, if you are interested.

Regards,

Pete

____________________
Peter M. Haverty, Ph.D.
Genentech, Inc.
phaverty@gene.com

On Mon, Jul 27, 2015 at 8:08 AM, Carlo Baldassi <notifications@github.com>
wrote:

> In src/ArgParse.jl
> <https://github.com/carlobaldassi/ArgParse.jl/pull/14#discussion_r35546293>
> :
>
> > @@ -1,10 +1,9 @@
> > -isdefined(:OptionsMod) || require("Options")
> > +isdefined(:OptionsMod) || @eval import Options
>
> Ok it's probably safer to use @eval (actually in JuliaLang/julia#12233
> <https://github.com/JuliaLang/julia/issues/12233> Jeff later suggested eval(Main,
> :(import Options))) even though I'm not sure exacly what would be
> different in this case.
>
> About Options being necessary, see #13
> <https://github.com/carlobaldassi/ArgParse.jl/issues/13>
>
> The new warning is indeed annoying, this issue is JuliaLang/Options.jl#4
> <https://github.com/JuliaLang/Options.jl/issues/4>.
>
> Compat is needed for startswith and other changes, see e.g. 996d7be
> <https://github.com/carlobaldassi/ArgParse.jl/commit/996d7be91142731edc07ab10eead324201cf936f>
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/carlobaldassi/ArgParse.jl/pull/14/files#r35546293>.
>